### DISCUSSION TITLE
第一次使用小白请教，谢谢

### DISCUSSION THREAD

**Discussion by hhwhtl** [SeaHOH/GotoX#135] [一般]
之前一直使用XX_Net，最近不太好用，看到GotoX，进来学习一下。
1.在使用XX_Net时部署过服务端，不知道是否还要再部署一次。运行uploader.bat后，提示警告：检测到本机没有在指定端口监听的 HTTP 代理 (8087) 或 SOCKS5 代理 (1080)， 建议先启动 GotoX 客户端或者其它代理，并根据代理类型设定监听的端口。请问如何操作？
2.运行GotoX 客户端后，选择自动代理，提示读取系统当前 DNS 设置失败，不知道如何操作。
谢谢！

**Comment by SeaHOH** [comment]
我不太擅长回答如何如何的问题。
第一点请阅读 readme，警告提示就是字面意思；
第二点如不影响使用请无视。

如果真心想使用，请先阅读一遍 readme、wiki 以及配置文件注释。
还有提问前的说明可能你也没注意，请更新到主分支。

**Comment by hhwhtl** [comment]
谢谢答复，我再试一试
